reporter/src/main/kotlin/reporters/ReportTableModelMapper.kt

```diff
@@ -207,7 +207,13 @@ class ReportTableModelMapper(private val resolutionProvider: ResolutionProvider)
 
         val evaluatorIssues = ortResult.evaluator?.let {
             it.errors.map { it.toResolvableEvaluatorIssue() }
-        } ?: emptyList()
+        }?.sortedWith(compareBy(
+                { it.isResolved },
```

Comment:
This means that resolved issues get displayed first, right? Shouldn't we rather display them last as these do not require any action anymore?

Reply:
No, they are displayed last, by default `true > false` and order is ascending.